gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -1000,7 +1000,7 @@ def create_override(
         Arguments:
             text: Text of the new override
             nvt_id: OID of the nvt to which override applies
-            seconds_active: Seconds override will be active. -1 on always, 0 off
+            days_active: Days override will be active. -1 on always, 0 off
```

Comment:
It already says Days?

Reply:
Sorry, thought I was looking at the "Files changed" tab when I had only the first patch open when I wrote my review.
I've dismissed it now.